### ISSUE TITLE
Missing conjugate of V and indexing issue 

### ISSUE THREAD

**Issue by wuziyou** [kunert/py-optDMD#1]
Hello James, 

I'm working on getting dmd modes for a noisy 3d real linear system, with a real and complex pair of eigenvalues. The observations are monomials. I found MATLAB version of optDMD by Travis Askham gave a different result from this implementation under exactly the same input. 

I think the code line 112 and 238 are missing a conjugacy of the V matrix. In line 153,168,194, there's probably an index mismatch of delta and alpha, I think it shall be `delta0[jpvt[:ia]]=delta0[range(ia)].` 

With these modifications I was able to get matching results from the MATLAB version. 

Best,
Ziyou

**Comment by klapo** [comment]
I got identical solutions between my fork of the optDMD and the reference MATLAB code. Is this still an issue? I would love to fix it if it is.

**Comment by wuziyou** [comment]
Yes, my test cases with complex eigenvalues had different results from the MATLAB code.

**Comment by klapo** [comment]
Sorry, for not communicating what I meant. Is it still an issue if you use [my fork](https://github.com/klapo/pyoptDMD)? I had identical results for a simple 2D case.
